backend/src/mailer/text/accountActivation.ts

```diff
@@ -0,0 +1,16 @@
+export const accountActivation = {
+  de: {
+    subject: 'Gradido: E-Mail Überprüfung',
+    text: (data: { link: string; firstName: string; lastName: string; email: string }): string =>
+      `Hallo ${data.firstName} ${data.lastName},
+
+Deine EMail wurde soeben bei Gradido registriert.
+
+Klicke bitte auf diesen Link, um die Registrierung abzuschließen und dein Gradido-Konto zu aktivieren:
+${data.link}
+oder kopiere den obigen Link in dein Browserfenster.
+
+Mit freundlichen Grüßen,
+dein Gradido-Team`,
+  },
+}
```

Comment:
I think this is a good solution, tho i would have chosen a different file for each language. This would make editing easier with multiple languages. Furthermore I would put the emails in a subfolder each:
```
mailer
| - accountActivation
    | - de.json
    | - en.json
    | - fr.json
| - transactionReceived
    | - de.json
    | - ....
```

If more subfolders but emails are required in `mailer/`, I would make another subfolder like `emails`.

Furthermore I would have chosen `mails` instead of `mailer` but I guess thats a matter of taste